src/components/main/note/edit-view/NoteEditView.js

```diff
@@ -26,23 +28,71 @@ const NoteEditView = ({ note, id }) => {
   const { windowDimensions, position } = note;
   const { innerHeight, innerWidth } = windowDimensions;
   const { pageX, pageY } = position;
+  const [ title, setTitle ] = useState(note.title);
+  const [ description, setDescription ] = useState(note.description);
+  const [ color, setColor ] = useState(note.color);
 
-  const { color } = useSelector(({ canvas }) => canvas.noteBeingEdited.data);
+  const { setNoteBeingEditedId } = useContext(CanvasContext);
+
+  const handleChangeText = (newTitle, newDescription) => {
+    setDescription(newDescription);
+    setTitle(newTitle);
+  }
```

Comment:
It just an idea, but maybe you wan to have a NoteContext? Or reuse the CanvasContext? I could easily imagine having saved the current note I am actually focused on in some context, so other components reuse these data / setters.
Though we might want to keep it simple, it's just an idea.

Reply:
I'll keep that in mind.